packages/fiori-generator-shared/test/read-me/expected/test01/README.md

```diff
@@ -0,0 +1,36 @@
+## Application Details
```

Comment:
as suggested I added snapshots - so we dont require expected folder any more

Reply:
I prefer we do not use snapshots but use files. Snapshots are difficult to review.

Reply:
Please add individual snapshot files if we are to change this. Large snapshot files are too difficult to review. Otherwise please use the file/folder snapshots.

Reply:
Resolving for now as we agreed to use file snapshot matchers in future